src/actions/pause.js

```diff
@@ -25,12 +25,12 @@ type CommandType = { type: string };
  */
 export function resumed() {
   return ({ dispatch, client }: ThunkArgs) => {
-    // dispatch(evaluateExpressions(null));
-
-    return dispatch({
+    dispatch({
       type: "RESUME",
       value: undefined
     });
+
+    dispatch(evaluateExpressions());
```

Comment:
a better pattern would be listening on the expressions reducer for a "resume" action. reusing evaluateExpressions here intertwines two responsibilities in an action.